spark-operator-api/src/main/java/org/apache/spark/k8s/operator/spec/JDKVersion.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.spec;
+
+public enum JDKVersion {
+  Java17,
+  Java21
```

Comment:
Just a question. Do we need to maintain in Apache Spark code?

If then, shall we follow `commons-lang3` style?
- https://github.com/apache/commons-lang/blob/056a2244e5ad37c0c3f38296cffb51a90b417967/src/main/java/org/apache/commons/lang3/JavaVersion.java#L144
```
JAVA_17,
JAVA_21
```

Reply:
Oh, wait. Is this supposed to be consistent with K8s style?

Reply:
Never mind this comment.